raft/node.go

```diff
@@ -140,20 +140,10 @@ type Peer struct {
 	Context []byte
 }
 
-// StartNode returns a new Node given a unique raft id, a list of raft peers, and
-// the election and heartbeat timeouts in units of ticks.
+// StartNode returns a new Node given configuration and a list of raft peers.
+// It ignores the given peer list in the given Config.
```

Comment:
Why does Config.Peers exist then? It sounds like it should be a separate parameter to newRaft instead of being a part of the public Config struct, since none of the public interfaces use it.